alephclient/tests/test_tasks.py

```diff
@@ -43,6 +42,27 @@ def test_write_entity(self, mocker):
         res = self.api.write_entity(collection_id, entity)
         assert res["id"] == 24
 
+
+    def test_delete_entity(self, mocker):
+        mocker.patch.object(self.api, "write_entity", return_value={"id": 24})
+        collection_id = 8
+        entity = {
+            "id": 24,
+            "schema": "Article",
+            "properties": {
+                "title": "",
+                "author": "",
+                "publishedAt": "",
+                "bodyText": "",
+            },
+        }
+
+        res = self.api.write_entity(collection_id, entity)
+        assert res['id'] == 24
+        dres = self.api.delete_entity(eid)
+        assert dres  == {}
```

Comment:
I'm not entirely sure what our `DELETE` endpoint returns, but shouldn't we (for completeness sake) try to get the entity with the same ID and expect a 404?